src/util-time.c

```diff
@@ -138,6 +138,20 @@ void CreateIsoTimeString (const struct timeval *ts, char *str, size_t size)
     }
 }
 
+void TimeToIsoTimeString (time_t time, char *str, size_t size)
```

Comment:
Is the result here the same as (or at least compatible to) the output by CreateIsoTimeString?